src/defs/policyRepresentation.ts

```diff
@@ -31,4 +31,5 @@ export default interface PolicyRepresentation {
   scopes?: string[];
   type?: string;
   users?: string[];
+  roles?: {id: string, required?: boolean}[];
```

Comment:
Can we split this type out into a separate interface? Where is this data structure documented/in the code?

Reply:
> Can we split this type out into a separate interface? Where is this data structure documented/in the code?

The roles Payload now has its own Interface and the rebase is done. Hopefully the CI now works!